test/images/image-util.sh

```diff
@@ -141,10 +141,10 @@ push() {
 # This function is for building the go code
 bin() {
   local arch_prefix=""
-  if [[ "${ARCH}" == "arm" ]]; then
+  if [[ "${ARCH:?}" == "arm" ]]; then
```

Comment:
```
In ./test/images/image-util.sh line 144:
  if [[ "${ARCH}" == "arm" ]]; then
         ^-----^ SC2153: Possible misspelling: ARCH may not be assigned, but arch is.
```

Reply:
Thanks for the comment.
`ARCH` is not misspelling. I modified it from "Exceptions" in https://github.com/koalaman/shellcheck/wiki/SC2153.
Should we avoid it with other way?

Reply:
Thanks. I have modified it.